integration-tests/src/test/java/org/wildfly/prospero/it/cli/CloneTest.java

```diff
@@ -0,0 +1,170 @@
+          )
+                .execute()
+                .assertReturnCode(ReturnCodes.SUCCESS);
+        checkMetaData(InstallationMetadata.importMetadata(exportPath));
+    }
+
+    private void mockInstallationMeta(Path installDir) throws Exception {
+        final Path manifestPath = temp.newFile().toPath();
+        final Path provisionConfig = temp.newFile().toPath();
+        MetadataTestUtils.copyManifest("channels/wfcore-19-base.yaml", manifestPath);
+        MetadataTestUtils.prepareProvisionConfig(provisionConfig, List.of(manifestPath.toUri().toURL()));
+        install(installDir, provisionConfig);
+    }
+
+    private void install(Path installDir, Path provisionConfig) throws Exception {
+        ExecutionUtils.prosperoExecution(CliConstants.Commands.INSTALL,
+                        CliConstants.PROVISION_CONFIG, provisionConfig.toString(),
+                        CliConstants.FPL, "wildfly-core@maven(org.jboss.universe:community-universe):19.0",
+                        CliConstants.DIR, installDir.toString())
+                .withTimeLimit(10, TimeUnit.MINUTES)
+                .execute()
+                .assertReturnCode(ReturnCodes.SUCCESS);
+    }
+
+    private void checkMetaData(InstallationMetadata metadata) {
+        try (InstallationMetadata meta = metadata) {
+            ChannelManifest channelManifest = meta.getManifest();
+            ProsperoConfig prosperoConfig = meta.getProsperoConfig();
+            ProvisioningConfig provisionedConfig = meta.getGalleonProvisioningConfig();
+            // check channelManifest
+            assertEquals(ChannelManifestMapper.CURRENT_SCHEMA_VERSION, channelManifest.getSchemaVersion());
+            assertNull(channelManifest.getDescription());
+            assertEquals(164, channelManifest.getStreams().size());
+            assertTrue(channelManifest.getStreams().stream()
+              .anyMatch(m -> m.getGroupId().equals("io.undertow")
+                && m.getArtifactId().equals("undertow-core")
+                && m.getVersion().equals("2.2.17.Final")));
+            assertTrue(channelManifest.getStreams().stream()
+              .anyMatch(m -> m.getGroupId().equals("org.jboss.xnio")
+                && m.getArtifactId().equals("xnio-nio")
+                && m.getVersion().equals("3.8.7.Final")));
+            assertTrue(channelManifest.getStreams().stream()
+              .anyMatch(m -> m.getGroupId().equals("org.wildfly.core")
+                && m.getArtifactId().equals("wildfly-cli")
+                && m.getVersion().equals("19.0.0.Beta11")));
+
+            // check prosperoConfig
+            assertEquals(1, prosperoConfig.getChannels().size());
+            Channel channel = prosperoConfig.getChannels().get(0);
+            assertEquals("channel-0", channel.getName());
+            assertEquals(3, channel.getRepositories().size());
+            channel.getRepositories().stream().anyMatch(p -> p.getId().equals("maven-central"));
```

Comment:
missing `assertTrue`?

also AssertJ Assertions.assertThat has really nice API for testing collections, that can become something like:
```
 assertThat(channel.getRepositories())
  .map(Repository::getId())
  .containsExactly("maven-central", "nexus", maven-redhat-ga");
 ```